src/main/webapp/styles/myprofile.css

```diff
@@ -0,0 +1,44 @@
+.button {
+  width: 200px;
+  height: 50px;
+}
+
+.details-container {
```

Comment:
As mentioned in a previous CL, can we keep this alphabetically-arranged to make it easier to keep track?

Reply:
on it!

Reply:
thanks! :)